source/index.html.md.erb

```diff
@@ -5,7 +5,7 @@ weight: 0
 
 # Introduction
 
-This is REST API documentation for the Department for Education’s new "Apply for postgraduate teacher training" service.
+This is JSON API documentation for the Department for Education (DfE)’s new "Apply for postgraduate teacher training" service.
```

Comment:
IMO, it is better to specify the format supported (Json) in a different line, 
and put 
`This is the API documentation ...`
in this line